digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/MassTransitStopService.scala

```diff
@@ -454,5 +624,11 @@ trait MassTransitStopService extends PointAssetOperations {
            where id = $id
       """.execute
   }
+
+  def expireMassTransitStop(s: String, id: Long) = {

```

Comment:
Tierekisteri is not synchronized on this. Should we signal the "valid to date" here as well to Tierekisteri?